tests/end-to-end/external-solution-integration/resourceskit/app_connector_client.go

```diff
@@ -0,0 +1,92 @@
+package resourceskit
+
+import (
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	acv1 "github.com/kyma-project/kyma/components/application-operator/pkg/apis/applicationconnector/v1alpha1"
+	verac "github.com/kyma-project/kyma/components/application-operator/pkg/client/clientset/versioned"
+	trv1 "github.com/kyma-project/kyma/components/connection-token-handler/pkg/apis/applicationconnector/v1alpha1"
+	vertr "github.com/kyma-project/kyma/components/connection-token-handler/pkg/client/clientset/versioned"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/consts"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+type AppConnectorClient interface {
+	CreateTokenRequest() (*trv1.TokenRequest, error)
+	GetTokenRequest() (*trv1.TokenRequest, error)
+	DeleteTokenRequest() error
+	CreateDummyApplication(skipInstallation bool) (*acv1.Application, error)
+	DeleteApplication() error
+	GetApplication() (*acv1.Application, error)
+}
+
+type appConnectorClient struct {
+	tokenClient       *vertr.Clientset
```

Comment:
Here the generated client is used, but you have also your own implementation which is used in TestSuite. Please decide and use one.

Reply:
I am not sure @janmedrek  do you have some idea here ?